frontend/src/GlobalStyles.js

```diff
@@ -0,0 +1,10 @@
+import {createGlobalStyle} from 'styled-components';
+
+export default createGlobalStyle`
+    * {
+        margin: 0;
```

Comment:
In case you realize you are often setting padding to 0 in some other elements, you can consider to set it here globally to 0 and then just adding it where you need it. But for now I think it is not necessary.